src/plugins/stepsform/stepsform-en.hbs

```diff
@@ -0,0 +1,106 @@
+				<div>
+					<div class="form-group">
+						<label for="preferredNumber" class="required">Preferred: <span class="field-name">Phone Number</span></label>
+						<input class="form-control" id="preferredNumber" name="preferredNumber" type="tel" data-rule-phoneUS="true" required="required" />
+					</div>
+					<div class="form-group">
+						<label for="secondaryNumber">Secondary: <span class="field-name">Phone Number</span></label>
+						<input class="form-control" id="secondaryNumber" name="secondaryNumber" type="tel" data-rule-phoneUS="true" />
+					</div>
+					<div class="form-group">
+						<label for="email"><span class="field-name">Email Address</span> (test@domain)</label>
+						<input class="form-control" id="email" name="email" type="email" />
+					</div>
+				</div>
+			</fieldset>
+			<fieldset id="questionaire">
+				<legend>Ethics Questionnaire</legend>
+				<div>
+					<small>For each of the following examples, indicate whether or not you feel that there is an ethical issue.</small>
+					<fieldset style="border:0;">
+						<legend><span class="field-name">Ethics 1</span>: You pay $10.00 bucks for a purchase, but receive change for $20.00 bucks. Would you keep the extra money?</legend>
+						<div class="radio">
+							<label for="defNot-1"><input type="radio" name="ethics1" value="defNot-1" id="defNot-1"/>&#160;Definitely Not</label>&#160;
+							<label for="probNot-1"><input type="radio" name="ethics1" value="probNot-1" id="probNot-1"/>&#160;Probably Not</label>&#160;
+							<label for="maybe-1"><input type="radio" name="ethics1" value="maybe-1" id="maybe-1"/>&#160;Maybe (Not Sure)</label>&#160;
+							<label for="probably-1"><input type="radio" name="ethics1" value="probably-1" id="probably-1"/>&#160;Probably</label>&#160;
+							<label for="definitely-1"><input type="radio" name="ethics1" value="definitely-1" id="definitely-1"/>&#160;Definitely</label>
+						</div>
+					</fieldset>
+					<fieldset style="border:0;">
+						<legend><span class="field-name">Ethics 2</span>: You need a software, but can't wait for office to make a decision. Against license restrictions would you download it?</legend>
+						<div class="radio">
+							<label for="defNot-2"><input type="radio" name="ethics2" value="defNot-2" id="defNot-2"/>&#160;Definitely Not</label>&#160;
+							<label for="probNot-2"><input type="radio" name="ethics2" value="probNot-2" id="probNot-2"/>&#160;Probably Not</label>&#160;
+							<label for="maybe-2"><input type="radio" name="ethics2" value="maybe-2" id="maybe-2"/>&#160;Maybe (Not Sure)</label>&#160;
+							<label for="probably-2"><input type="radio" name="ethics2" value="probably-2" id="probably-2"/>&#160;Probably</label>&#160;
+							<label for="definitely-2"><input type="radio" name="ethics2" value="definitely-2" id="definitely-2"/>&#160;Definitely</label>
+						</div>
+					</fieldset>
+				</div>
+			</fieldset>
+			<input type="submit" name="submit" value="Submit" class="btn btn-primary" />
```

Comment:
For some reason, after the plugin initialization that input is moved inside the preceding fieldset which is are not totally right from a sementic point of view.

Can you check to move all the navigation button outside the fieldset?

Reply:
resolved